content/docs/getting-started/introduction.md

```diff
@@ -1,29 +1,42 @@
 ---
-title: Tina is Not a CMS
+title: Introductory Tutorial
 id: introduction
 prev: null
-next: /docs/cms
+next: /docs/getting-started/cms-set-up
 ---
 
-Tina is a toolkit for building content management systems.
+Tina is a toolkit for building content management systems. This tutorial will **walk you through Tina's fundamental building blocks** by setting-up editing on a _very simple_ `create-react-app` [demo](https://github.com/tinacms/tina-intro-tutorial). To get the most from Tina, you should have a good **working knowledge of JavaScript and React**.
```

Comment:
```suggestion
Tina is a toolkit for building content management systems. This tutorial will walk you through Tina's fundamental building blocks by setting up editing on a basic `create-react-app` [demo](https://github.com/tinacms/tina-intro-tutorial). **To get the most from Tina, you should have a good working knowledge of JavaScript and React**.
```

motivations for this change:

1. Avoiding [emphasis fatigue](https://tinacms.org/docs/contributing/style-guide#formatting)
2. Trying to avoid referring to things as "simple" or "easy"